Source/WebCore/Modules/webauthn/AuthenticatorCoordinator.cpp

```diff
@@ -191,6 +207,17 @@ void AuthenticatorCoordinator::create(const Document& document, CredentialCreati
         ASSERT(!exception.message.isNull());
         promise.reject(exception.toException());
     };
+
+    if (m_isCancelling) {
+        m_queuedRequest = [weakThis = WeakPtr { *this }, weakFrame = WeakPtr { *frame }, clientDataJsonHash = WTFMove(clientDataJsonHash), createOptions = WTFMove(createOptions), callback = WTFMove(callback)]() mutable {
+            if (!weakThis || !weakFrame)
+                return;
+            const auto& options = createOptions.publicKey.value();
+            const auto& callerOrigin = weakFrame->document()->securityOrigin();
+            weakThis->m_client->makeCredential(*weakFrame, callerOrigin, clientDataJsonHash, options, createOptions.mediation, WTFMove(callback));
```

Comment:
Please store the Frame in a RefPtr before passing it as parameter here, as per our smart pointer guidelines:
```
RefPtr frame = weakFrame.get();
if (!frame)
    return;
weakThis->m_client->makeCredential(*frame, callerOrigin, clientDataJsonHash, options, createOptions.mediation, WTFMove(callback));
```